File: Pod/Classes/FNKPieSectionData.m
//
//  FNKPieSectionData.m
//  Pods
//
//  Created by Phillip Connaughton on 1/11/15.
//
//

#import "FNKPieSectionData.h"

@implementation FNKPieSectionData

+(FNKPieSectionData*) pieSectionWithName:(NSString*)name color:(UIColor*)color percentage:(CGFloat)percentage
{
    FNKPieSectionData* data = [[FNKPieSectionData alloc] init];
    data.name = name;
    data.color = color;
    data.percentage = percentage;
    return data;
}

@end

File: Pod/Classes/FNKPieSectionData.swift
//
//  FNKPieSectionData.swift
//  Pods
//
//  Created by Phillip Connaughton on 11/21/15.
//
//

import Foundation

class FNKPieSectionData
{
    let percentage: Float
    let name: String
    let color: UIColor
    let sliceValue: Float
    
    var slice: CAShapeLayer?
    var sliceLabel: UILabel?
    
    init(name: String, color: UIColor, percentage: Float, sliceValue: Float)
    {
        self.name = name
        self.color = color
        self.percentage = percentage
        self.sliceValue = sliceValue
    }

}